packages/bsync/src/purchases/purchasesClient.ts

```diff
@@ -0,0 +1,269 @@
+import { Subscription } from '../proto/bsync_pb'
+import { RcSubscription, RevenueCatClient } from './revenueCatClient'
+
+type PlatformId = 'web' | 'ios' | 'android'
+
+type SubscriptionGroupId = 'core'
+
+type SubscriptionOfferingId = 'coreMonthly' | 'coreAnnual'
```

Comment:
Ya know what might be good? Calling these `core:monthly` and `core:annual`, just to hammer home the relationship. Not a big deal, your call if you wanna do that right now.